xc/xc7/tests/soc/litex/mini/CMakeLists.txt

```diff
@@ -1,18 +1,38 @@
-add_file_target(FILE mem.init)
-add_file_target(FILE mem_1.init)
-add_file_target(FILE mem_2.init)
+add_file_target(FILE arty_clocks.xdc)
 
-add_fpga_target(
+set(SOURCES
+  top.v
+  top.pcf
+  top.sdc
+  mem.init
+  mem_1.init
+  mem_2.init
+)
+
+add_litex_test(
   NAME minilitex_arty
+  LITEX_DIR arty_soc
+  LITEX_BOARD a7-35
+  LITEX_SOURCES ${SOURCES}
+  EXTERNAL_SOURCES ${VEXRISCV}
   BOARD arty-full
-  SOURCES
-    minilitex_arty.v
-    VexRiscv_Lite.v
-  INPUT_IO_FILE arty.pcf
-  INPUT_XDC_FILE minilitex.xdc
+  GENERATE_SCRIPT ${GENERATE_LITEX}
+  FIXUP_SCRIPT ${FIXUP_XDC}
+  FLAGS
+    --toolchain symbiflow
```

Comment:
What does `--toolchain symbiflow` do?  I noticed that `xc/xc7/tests/soc/litex/mini_ddr/CMakeLists.txt` doesn't use `--toolchain symbiflow`.

Reply:
This was related to the fact that the PLL generate extra clocks and BUFG which are not used, ending up in clocks generated in the SDC, but they are absent in the cleaned circuit in VPR, therefore the SDC commands for the inexistent generate an error

Reply:
Please add a row into the burndown to address this issue.  If VPR sweeps a netlist, and an SDC that was valid looks invalid, that needs to not be an error.